File: iOS/Martinshare/Martinshare/LoginProtocol.swift
//
//  LoginProtocol.swift
//  Martinshare
//
//  Created by Modestas Valauskas on 25.05.15.
//  Copyright (c) 2015 ModestasV Studios. All rights reserved.
//

import Foundation

protocol LoginProtocol {

    func startedLogingIn()
    func rightCredentials()
    func wrongCredentials()
    func noInternetConnection()
    func unknownError(_ string: String)
    
}

File: Android/Martinshare/app/src/main/java/mobile/martinshare/com/martinshare/API/protocols/LoginProtocol.java
package mobile.martinshare.com.martinshare.API.protocols;

/**
 * Created by Modestas Valauskas on 03.06.2015.
 */
public interface LoginProtocol {
    public void startedLogingIn();
    public void rightCredentials();
    public void wrongCredentials();
    public void noInternetConnection();
}
